app/workers/rapid_ready_for_decision/form526_base_job.rb

```diff
@@ -146,7 +146,7 @@ def upload_pdf(form526_submission, pdf)
     end
 
     def set_special_issue(form526_submission)
-      return if release_pdf?(form526_submission)
+      return unless release_pdf?(form526_submission)
```

Comment:
FYI: Unless you do it here or suggest otherwise, I plan to move the condition check outside of this method, i.e., so line 65 will become:
```ruby
set_special_issue(form526_submission) if Flipper.enabled?(:rrd_add_special_issue) && release_pdf?(form526_submission)
```